src/styles/Stars.scss

```diff
@@ -0,0 +1,29 @@
+  .card {
+
+  .stars {
+    width: 96px;
+    height: 16px;
+    display: flex;
+  }
+
+  .stars__star {
```

Comment:
- Stars should be a separate block, not only inside the .card
- Use SCSS nesting